sos/plugins/openstack_containerlogs.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2019 Red Hat, Inc., Cédric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os
+
+
+class OpenstackContainerLog(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """OpenStack container logs
+    """
+    plugin_name = "openstack_containerlog"
+    profiles = ('openstack', 'openstack_controller', 'openstack_compute')
```

Comment:
Looks like my previous review got cut off somehow. Anyways...

See my notes about other components using `/var/log/containers` - we don't necessarily want to limit this plugin's scope to OpenStack.

At the same time, this plugin _should_ have some form of enablement gating like packages or services.

Reply:
Hello! yeah, sorry for the "hard cut", I saw the notification from the bot and pushed an update right after that.

regarding the profiles: I wasn't aware that location was also used by other products - so of course we will need to update that listing. I can already check if, among the editions I've done in this commit, some plugins weren't related to openstack. To be honest I didn't think about that :/.
That would be a good start.

Reply:
hmm. apparently, there's also the "services" group for MongoDB (as a first quick pass).

We might actually "just" check if the /var/log/containers directory exists and, if so, enable it... Would that kind of solution be OK? Is there already a plugin using this capability?

Reply:
You can use the `files` class member for `Plugin`, yeah. It ends up just doing an `os.path.exists()` on anything specified.

Reply:
Perfect, applied in last version with your comment :). Lemme know if I missed something, but.. Should be fine. Hopefully.

Thank you for your time!